monai/inferers/utils.py

```
@@ -33,12 +33,51 @@
+
 tqdm, _ = optional_import("tqdm", name="tqdm")
 _nearest_mode = "nearest-exact"
 
 __all__ = ["sliding_window_inference"]
 
 
+
+def assert_channel_first(
+    t: torch.Tensor,
+    name: str,
+    num_classes: Optional[int] = None,
```

Ruff (0.14.6)

Undefined name `Optional`

(F821)